_posts/2022-12-06-integrating-auth0-with-react.md

```diff
@@ -0,0 +1,307 @@
+---
+layout: post
+font: serif
+title: Integrating Auth0 Authentication into React
+date: 2022-12-06
+highlight: monokai
+author: Chris Howard
+image: img/blog/integrating-auth0-with-react/banner.jpg
+tags: Auth0 react authentication typescript
+description: 
+---
+
+Authentication and Authorization is objectively the most important part of any application, So why is it so often left to the end of development cycles or implemented in a slap dash manner?
+
+It may be due to the difficulty in creating authentication systems and implementing them into our applications, but with the tools and services available to developers it has become easier and easier to create a robust, secure and easy to use system. So let's not wait and get started on integrating [Auth0](https://auth0.com/) into a new React application!
+
+<br />
+
+## Contents
+
+- [Why Auth0?](#why-auth0)
+- [Create Auth0 Application](#create-auth0-application)
+- [Integration into React](#integrate-with-react)
+    * [Login](#but-wait-how-do-users-login)
+    * [Logout](#great-now-how-about-logout)
+- [Conclusion](#conclusion)
+
+## Why Auth0?
+
+Auth0 is one of the most highly reputed company working in this space. Why build your own service when you can get the years of experience and knowledge dedicated to making sure your users login data is safe?
+
+Auth0 provides Authentication, Authorization, Social Logins, MFA, Analytics and lots more, And to make it all the sweeter, has amazing developer documentation. All of this completely free if you have under 7000 users!
+
+## Create Auth0 Application
+
+Let's start with create a new Auth0 Application
+
+Select Applications > Applications from the side menu (or alternatively the Create Application on the Getting Started page)
```

Comment:
You're missing context here = where should I be?

In an application?  
On a web page?  Which one?

Reply:
Added some additional context

> Login to [Auth0](https://auth0.com/) (or create a new account if you haven't already) and navigate to your [Dashboard](https://manage.auth0.com/dashboard)